stable_baselines3/common/base_class.py

```diff
@@ -446,7 +446,11 @@ def _setup_learn(
 
         # Avoid resetting the environment when calling ``.learn()`` consecutive times
         if reset_num_timesteps or self._last_obs is None:
-            self._last_obs = self.env.reset()  # pytype: disable=annotation-type-mismatch
+            # TODO: should the info dict be saved somewhere?
```

Comment:
we could (even though here I'm not sure if it's helpful) and I think that will be my plan for the vec env, instead of changing the api, save it in a variable that the user can retrieve.